tests/tabular_output/test_tabulate_adapter.py

```diff
@@ -19,3 +19,20 @@ def test_tabulate_wrapper():
         | abc       |        1 |
         | d         |      456 |
         +-----------+----------+''')
+
+
+def test_markup_format():
+    """Test that markup formats do not have number align or string align."""
```

Comment:
Yeah, lI think you're right that we don't want column alignment for the markup formats like HTML.

Is there another tabulate column alignment besides `stralign` and `numalign`?